R/mpf_exporters.R

```diff
@@ -388,14 +398,17 @@ mpf_to_rtf <- function(mpf,
 #' @export
 export_as_rtf <- function(x,
                           file = NULL,
+                          # colwidths = propose_column_widths(matrix_form(x, TRUE, fontspec = fontspec), fontspec = fontspec), # nolint
                           colwidths = NULL,
                           page_type = "letter",
                           pg_width = page_dim(page_type)[if (landscape) 2 else 1],
                           pg_height = page_dim(page_type)[if (landscape) 1 else 2],
                           landscape = FALSE,
                           margins = c(bottom = .5, left = .75, top = .5, right = .75),
-                          font_size = 8,
                           font_family = "Courier",
+                          font_size = 8,
+                          lineheight = 1,
```

Comment:
different lineheight?

Reply:
well, `export_as_rtf` is kinda a mess, but the headline here is that historically the pdf exporter was using the default within grid if lineheight isn't set, which seems to be 1.2, but other places I think the default we set has been 1.

Probably need to unify but that is a potentially breaking change so my PR just leaves the mismatching defaults as they are for lineheight

Reply:
I understand. We will have to sit down at a certain point and set more general defaults

Reply:
We had to add a way to set the hsep globally because the continuous hyphen was problematic for non-UTF8 system (dont ask me why they are a thing). Just wondering if it would make sense to have an object with defaults that lives in the package. It is of course a discussion for another time but let me know if you think it could make sense for the future